modern/gcc/Dockerfile

```diff
@@ -61,8 +61,6 @@ RUN sudo rm -rf /usr/lib/gcc/x86_64-linux-gnu/* \
     && sudo cp -a /tmp/install/bin/* /usr/local/bin/ \
     && sudo rm -rf /tmp/install \
     && sudo update-alternatives --install /usr/local/bin/cc cc /usr/local/bin/gcc 100 \
```

Comment:
Why do we need this one? Is it not in the GCC bundle?

Reply:
cc is the only one not available. However, it's the same as gcc. I used the official docker image `gcc:11` to compare those binaries and `diff cc gcc` produces equal binaries. The cpp is different indeed, it's not a link, neither gcc, it's an unique binary.

Reply:
```
$ docker run --rm -ti gcc:11 diff -s /usr/bin/cc /usr/local/bin/gcc
Files /usr/bin/cc and /usr/local/bin/gcc are identical
```

Reply:
```
$ docker run --rm -ti gcc:11 diff -s /usr/local/bin/cpp /usr/local/bin/gcc
Binary files /usr/local/bin/cpp and /usr/local/bin/gcc differ
```

Reply:
what about /usr/bin/c++?

Reply:
it's already there. It will use /usr/local/bin/c++